check_source.sh

```diff
@@ -10,6 +10,19 @@ if [ -f check-source.pl ] ; then
   perl check-source.pl || exit 1
 fi
 
+if [ -f ./updatemakes.sh ] ; then
+  echo "checking makefiles..."
+  beforeupdate=$(grep '' makefile*)
+  ./updatemakes.sh
```

Comment:
I like that a lot better, but I still don't like the fact that the files get now modified on disk when you run this (I especially dislike that the filename lies and says 'check' and then does 'modify' :-) )